### PR TITLE
Tackles update jg

### PR DESCRIPTION
**Pull request by janegreene**
Updated season tackles methods to compare on whole season. Currently passes spec harness.

### CODE REVIEW

File: lib/season_repository.rb [hunk 1]
```diff
@@ -103,20 +103,21 @@ def most_tackles(season_id)
   games_in_the_season.each do |season_game|
     @game_team_collection.each do |game_team|
       if game_team.game_id == season_game
-        if game_team.tackles > highest_tackles
-          highest_tackles = game_team.tackles
-          highest_team_id = game_team.team_id
+        if tackles_team.key?(game_team.team_id)
```
Comment (janegreene) [inline lib/season_repository.rb:106]: Changed this to now make a hash

File: lib/season_repository.rb [hunk 2]
```diff
@@ -143,37 +146,5 @@ def find_team_id(id)
   named_team
 end
 
-def most_accurate_team(season_id)
```
Comment (EricLarson2020) [inline lib/season_repository.rb:146]: Good work changing this

File: lib/season_repository.rb [hunk 3]
```diff
@@ -103,20 +103,21 @@ def most_tackles(season_id)
   games_in_the_season.each do |season_game|
     @game_team_collection.each do |game_team|
       if game_team.game_id == season_game
-        if game_team.tackles > highest_tackles
```
Comment (EricLarson2020) [inline lib/season_repository.rb:106]: Good work with the nested iterations

**Review by EricLarson2020 — APPROVED** [review]
Nice Job!

**Pull request merged**